app/src/main/java/org/thoughtcrime/securesms/onboarding/messagenotifications/MessageNotifications.kt

```diff
@@ -55,29 +57,36 @@ internal fun MessageNotificationsScreen(
 
     Column {
         Spacer(Modifier.weight(1f))
-
         Column(modifier = Modifier.padding(horizontal = LocalDimensions.current.mediumSpacing)) {
             Text(stringResource(R.string.notificationsMessage), style = LocalType.current.h4)
             Spacer(Modifier.height(LocalDimensions.current.smallSpacing))
-            Text(stringResource(R.string.onboardingMessageNotificationExplaination), style = LocalType.current.base)
+            Text(
+                Phrase.from(stringResource(R.string.onboardingMessageNotificationExplanation))
+                    .put(APP_NAME_KEY, stringResource(R.string.app_name))
+                    .format().toString(),
+                style = LocalType.current.base
+            )
             Spacer(Modifier.height(LocalDimensions.current.spacing))
         }
 
         NotificationRadioButton(
-            R.string.activity_pn_mode_fast_mode,
-            R.string.activity_pn_mode_fast_mode_explanation,
-            modifier = Modifier.contentDescription(R.string.AccessibilityId_fast_mode_notifications_button),
-            tag = R.string.activity_pn_mode_recommended_option_tag,
+            R.string.notificationsFastMode,
+            R.string.notificationsFastModeDescription,
+            modifier = Modifier.contentDescription(R.string.AccessibilityId_notificationsFastMode),
+            tag = R.string.recommended,
             checked = state.pushEnabled,
             onClick = { setEnabled(true) }
         )
 
         // spacing between buttons is provided by ripple/downstate of NotificationRadioButton
 
+        val txt = Phrase.from(stringResource(R.string.onboardingMessageNotificationExplanation))
```

Comment:
This `txt` property is never used

Reply:
This is now used in the version of `NotificationRadioButton` that takes strings for title and explanations rather than resource IDs.